packages/core/src/browser/style/variables-bright.useable.css

```diff
@@ -256,4 +256,7 @@ is not optimized for dense, information rich UIs.
   --theia-ansi-cyan-background-color: darkcyan;
   --theia-ansi-white-background-color: #BDBDBD;
 
+  /* Terminal */
+  --theia-terminal-selection-background-color: #00000040;
```

Comment:
Core should not be aware about the terminal.

please introduce it in terminal extension, see how it is done fork debug 
extension, i.e. `debug-dark.useable.css` and `debug-bright.useable.css``